flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/config/MySqlSourceOptions.java

```diff
@@ -211,4 +213,14 @@ public class MySqlSourceOptions {
                     .defaultValue(false)
                     .withDescription(
                             "Whether capture the scan the newly added tables or not, by default is false.");
+
+    @Experimental
+    public static final ConfigOption<String> SCAN_INCREMENTAL_SNAPSHOT_CHUNK_KEY =
+            ConfigOptions.key("scan.incremental.snapshot.chunk.key")
```

Comment:
```suggestion
            ConfigOptions.key("scan.incremental.snapshot.chunk.key-column")
```
I think `key-column` is more clear than `key`, just like we use `value.primary-key-columns` in kafka source